src/Firebase/FirebaseStore.tsx

```diff
@@ -24,8 +24,6 @@ export const updateUserInFirestore = async (
         fullName: fullName || user.displayName,
         userID: user.uid,
```

Comment:
since the document ref id is saved as the user's uid, it doesn't need to be declared again as userID